modules/web/src/app/wizard/step/cluster/component.ts

```diff
@@ -499,6 +448,167 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
+      [Controls.Labels]: this._builder.control(null),
+      [Controls.PodNodeSelectorAdmissionPluginConfig]: this._builder.control(null),
+      [Controls.SSHKeys]: this._builder.control(this._clusterSpecService?.sshKeys ?? ''),
+      [Controls.IPFamily]: this._builder.control(clusterSpec?.clusterNetwork?.ipFamily ?? IPFamily.IPv4),
+      [Controls.ProxyMode]: this._builder.control(clusterSpec?.clusterNetwork?.proxyMode ?? ''),
+      [Controls.CNIPlugin]: this._builder.control(clusterSpec?.cniPlugin?.type ?? CNIPlugin.Canal),
+      [Controls.CNIPluginVersion]: this._builder.control(clusterSpec?.cniPlugin?.version ?? ''),
+      [Controls.IPv4CIDRMaskSize]: this._builder.control(clusterSpec?.clusterNetwork?.nodeCidrMaskSizeIPv4 ?? null),
+      [Controls.IPv6CIDRMaskSize]: this._builder.control(clusterSpec?.clusterNetwork?.nodeCidrMaskSizeIPv6 ?? null),
+      [Controls.NodeLocalDNSCache]: this._builder.control(
+        clusterSpec?.clusterNetwork?.nodeLocalDNSCacheEnabled ?? false
+      ),
+      [Controls.ExposeStrategy]: this._builder.control(clusterSpec?.exposeStrategy ?? null),
+      [Controls.APIServerAllowedIPRanges]: this._builder.control(
+        clusterSpec?.apiServerAllowedIPRanges?.cidrBlocks ?? null
+      ),
+      [Controls.IPv4PodsCIDR]: this._builder.control(NetworkRanges.ipv4CIDR(clusterSpec?.clusterNetwork?.pods) ?? '', [
+        IPV4_CIDR_PATTERN_VALIDATOR,
+        this._dualStackRequiredIfValidator(Controls.IPv6PodsCIDR),
+      ]),
+      [Controls.IPv6PodsCIDR]: this._builder.control(NetworkRanges.ipv6CIDR(clusterSpec?.clusterNetwork?.pods) ?? '', [
+        IPV6_CIDR_PATTERN_VALIDATOR,
+        this._dualStackRequiredIfValidator(Controls.IPv4PodsCIDR),
+      ]),
+      [Controls.IPv4ServicesCIDR]: this._builder.control(
+        NetworkRanges.ipv4CIDR(clusterSpec?.clusterNetwork?.services) ?? '',
+        [IPV4_CIDR_PATTERN_VALIDATOR, this._dualStackRequiredIfValidator(Controls.IPv6ServicesCIDR)]
+      ),
+      [Controls.IPv6ServicesCIDR]: this._builder.control(
+        NetworkRanges.ipv6CIDR(clusterSpec?.clusterNetwork?.services) ?? '',
+        [IPV6_CIDR_PATTERN_VALIDATOR, this._dualStackRequiredIfValidator(Controls.IPv4ServicesCIDR)]
+      ),
+      [Controls.IPv4AllowedIPRange]: this._builder.control(
+        this.isAllowedIPRangeSupported()
+          ? NetworkRanges.ipv4CIDR(this._getExtraCloudSpecOptions().nodePortsAllowedIPRanges)
+          : '',
+        [IPV4_CIDR_PATTERN_VALIDATOR, this._dualStackRequiredIfValidator(Controls.IPv6AllowedIPRange)]
+      ),
+      [Controls.IPv6AllowedIPRange]: this._builder.control(
+        this.isAllowedIPRangeSupported()
+          ? NetworkRanges.ipv6CIDR(this._getExtraCloudSpecOptions().nodePortsAllowedIPRanges)
+          : '',
+        [IPV6_CIDR_PATTERN_VALIDATOR, this._dualStackRequiredIfValidator(Controls.IPv4AllowedIPRange)]
+      ),
+    });
+  }
+
+  private _loadClusterDefaults(): void {
```

Comment:
We're not updating `labels` and `podNodeSelectorAdmissionPluginConfig` in this method, is it intentional?